pkg/core/pipeline/condition/main.go

```diff
@@ -36,15 +37,16 @@ type Config struct {
 	Postfix            string // Deprecated in favor of Transformers on 2021/01/3
 	Transformers       transformer.Transformers
 	Spec               interface{}
-	Scm                map[string]interface{}
-	SourceID           string `yaml:"sourceID"`
+	Scm                map[string]interface{} // Deprecated field on version [x.y.z]
```

Comment:
I suppose this is a placeholder until the final version number is determined?

Reply:
Yes so we can read existing configuration, load them in memory and then convert them so we can use it in updatecli

Reply:
I was speakig of the comment in this line, not sure I follow your response.

Reply:
Thanks for the clarification, indeed that was a placeholder until I identify the version that this PullRequest would land
```suggestion
	Scm                map[string]interface{} // Deprecated field on version [1.17.0]
```